### ISSUE TITLE
Possible config addition

### ISSUE THREAD

**Issue by Zaphara94** [mcMMO-Dev/mcMMO#3962]
Would be really nice if there was a config option to disable the new salvage mechanic, and go with a percent based system again, with the possibility to have 100% return.

**Comment by nossr50** [comment]
Salvage does allow for a chance to get 100% returns

**Comment by Zaphara94** [comment]
Yes, but I would like it to have the option to disable that "chance" and make it a 100% chance at a certain level to get 100% returns. Like just one config option to revert to the old system for salvage :x

**Comment by nossr50** [comment]
@Zaphara94 I'm currently rewriting Repair/Salvage in 2.2 and you will be able to specify the returns

**Comment by Zaphara94** [comment]
Awesome thanks <3 That'll make me very happy
